src/buildfile

```diff
@@ -37,17 +36,21 @@ cxx.libs = $intf_libs $extra_libs
 
 # Export options.
 #
-lib{benchmark*}:
+lib{benchmark}:
 {
-  cxx.export.poptions += "-I$out_root/include" "-I$src_root/include" $windows_additional_defines
+  # cxx.export.poptions += "-I$out_root/include" "-I$src_root/include" $windows_additional_defines
```

Comment:
I would drop this completely not to tempt anyone to uncomment it. This is definitely wrong on the conceptual level.

Reply:
Done